src/contentmanager.cpp

```diff
@@ -975,6 +1001,11 @@ QStringList ContentManager::getBookIds()
     }
 }
 
+QString ContentManager::getRemoteLibraryUrl() 
+{
+    return "https://library.kiwix.org";
+}
```

Comment:
Implement like [requested](https://github.com/kiwix/kiwix-desktop/pull/1110#discussion_r1601058698) in my previous review:

> `BASEURL` can be [overriden while debugging](https://github.com/kiwix/kiwix-desktop/commit/c10026a3a2616245f5ea47dedf3be436f5e07bc1). Please generate `BASEURL` as [this](https://github.com/kiwix/kiwix-desktop/commit/535675afdb0fce40121ccafa06909b61ad057895#diff-b143bce41389bc342b560b90019ea25d3a82243fab36352befb2be7401cc2b64R682-R701). In order to avoid introducing code duplication, please add a private helper function `ContentManager::getRemoteLibraryUrl()`.

Reply:
And then use `getRemoteLibraryUrl()` in `ContentManager::updateRemoteLibrary()` too

Reply:
Done